### PR TITLE
Skip vercel build on Renovate PRs

### PR DESCRIPTION
**Pull request by junlarsen**
image

See https://github.com/dotkom/monoweb/pull/686 for example

### CODE REVIEW

**Comment by kharann** [comment]
Perhaps add `turno-ignore` on the command too?

File: infra/modules/vercel-application/vercel.tf [hunk 1]
```diff
@@ -10,6 +10,9 @@ resource "vercel_project" "this" {
 
   build_command  = var.build_command
   root_directory = var.root_directory
+
+  # Do not run on renovate pull requests
+  ignore_command = "[ \"$VERCEL_GIT_COMMIT_AUTHOR_LOGIN\" == \"renovate[bot]\" ]"
```
Comment (kharann) [inline infra/modules/vercel-application/vercel.tf:15]: ignore_command = "[ \"$VERCEL_GIT_COMMIT_AUTHOR_LOGIN\" == \"renovate[bot]\" && turbo-ignore ]"

**Review by kharann — APPROVED** [review]
good other than that

File: infra/modules/vercel-application/vercel.tf (hunk 1 above)
Comment (junlarsen) [reply infra/modules/vercel-application/vercel.tf:15]: Perhaps npx turbo-ignore?

Comment (kharann) [reply infra/modules/vercel-application/vercel.tf:15]: Yeah probably. Thought you didn't need the npx

**Pull request merged**